### PR TITLE
Add NzbHydra 2

### PR DESCRIPTION
**Pull request by maxrogers**
_Motivation:_ Explain here what the reason for the pull request is.
_Linked issues:_ Optionally, add links to existing issues or other PR's

### Checklist
- [x] Build rule `all-supported` completed successfully
- [x] Package upgrade completed successfully
- [x] New installation of package completed successfully

This is a corrected pull request. I messed up when trying to squash the previous pull request (here: https://github.com/SynoCommunity/spksrc/pull/3260) This is the same as the other but it does update NzbHydra to v2.0.9

### CODE REVIEW

File: spk/nzbhydra2/Makefile [hunk 1]
```diff
@@ -0,0 +1,37 @@
+SPK_NAME = nzbhydra2
+SPK_VERS = 2.0.9
+SPK_REV = 1
+SPK_ICON = src/nzbhydra2.png
+
+DEPENDS = cross/$(SPK_NAME)
+SPK_DEPENDS = "Java8>8.0.151:git"
```
[the remark this answers is not in the document]
Comment (maxrogers) [reply spk/nzbhydra2/Makefile]: Yes, I have removed git dependency

**Comment by theotherp** [comment]
If run via the python wrapper script it should work on any machine where a JRE and python 2.7 is installed.

**Comment by maxrogers** [comment]
> If run via the python wrapper script it should work on any machine where a JRE and python 2.7 is installed.

I see. Thanks!

@ymartin59 The current package is directly running the compiled Java executable. This version currently requires x64. If we switch to the method @theotherp mentioned above (using the python wrapper) we may avoid the x64 limitation.

However, I think running the compiled java directly and limiting to only x64 might be a better decision. Nzbhydra2 uses a fair amount of resources (ram) and I worry about potential issues/reduction in performance when adding a python wrapper layer to the equation.

**Comment by hackgrid** [comment]
Is there an update to this? Would be great to have NzbHydra2 on Synology :-)

**Comment by funzzy1** [comment]
Hi,

I have created a repo with my spk.

You just need to download and install it. It is based on the nzbhydra spk from synocommunity.

I tested it on a DS415play NAS and it is working correctly:

[nzbhydra2-2.20.7.noarch.spk](https://github.com/funzzy1/nzbhydra2_spk/releases/tag/2.20.7)

**Comment by hackgrid** [comment]
> 
> 
> Hi,
> 
> I have created a repo with my spk.
> 
> You just need to download and install it. It is based on the nzbhydra spk from synocommunity.
> 
> I tested it on a DS415play NAS and it is working correctly:
> 
> [nzbhydra2-2.20.7.noarch.spk](https://github.com/funzzy1/nzbhydra2_spk/releases/tag/2.20.7)

Thanks, works great :-)

But it uses quite some CPU on my DS415+, is my NAS too old?

**Comment by funzzy1** [comment]
I have a 415play and I don't have problem.
415+ is more powerful, so it should work correctly.

**Comment by hgy59** [comment]
nzbhydra2 runs on x64 archs only and requires java.
As there is a docker image available and all x64 arch models support docker, it is better to use the docker version for nzbhydra2 on synology diskstations.

**Comment by hgy59** [comment]
closed due to inactivity

**Pull request closed without merging**